crates/autopilot/src/database/auction.rs

```diff
@@ -28,17 +28,6 @@ impl QuoteStoring for Postgres {
         Ok(id)
     }
 
-    async fn get(&self, id: QuoteId) -> Result<Option<QuoteData>> {
-        let _timer = super::Metrics::get()
-            .database_queries
-            .with_label_values(&["get_quote"])
-            .start_timer();
-
-        let mut ex = self.pool.acquire().await?;
-        let quote = database::quotes::get(&mut ex, id).await?;
```

Comment:
This function is used only in the tests now. It should be either moved to the tests section or `find` function can be used instead.